packages/design-system/src/components/_OcTableRow/_OcTableRow.vue

```diff
@@ -1,7 +1,6 @@
 <template>
   <tr
     ref="observerTarget"
-    tabindex="-1"
```

Comment:
I don't see a reason to set `tabindex="-1"` here. A `tr` can't be focused and we don't have a scenario where we want to focus it programmatically.